syntax = "proto3";

package relationalai.lqp.v1;

import "relationalai/lqp/v1/logic.proto";

message Fragment {
  FragmentId id = 1;
  repeated Declaration declarations = 2;
}

message FragmentId {
  bytes id = 1; // Variable-length identifier, up to 256 bits (32 bytes) or less
}
